libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1077,6 +1082,89 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info2[] = {
     // @User: Standard
     AP_GROUPINFO("FONT", 4, AP_OSD_Screen, font_index, 0),
 #endif
+
+#if AP_OSD_CRSF_PANELS_ENABLED
+    // @Param: CRSF_PWR_EN
+    // @DisplayName: CRSF_PWR_EN
+    // @Description: Displays the TX power when using the CRSF RC protocol
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: CRSF_PWR_X
+    // @DisplayName: CRSF_PWR_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 59
+
+    // @Param: CRSF_PWR_Y
+    // @DisplayName: CRSF_PWR_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 21
+    AP_SUBGROUPINFO(crsf_tx_power, "CRSF_PWR", 55, AP_OSD_Screen, AP_OSD_Setting),
+
+    // @Param: CRSFRSSI_EN
+    // @DisplayName: CRSFRSSI_EN
+    // @Description: Displays RC signal strength in dBm for CRSF
+    // @Values: 0:Disabled,1:Enabled
+
+    // @Param: CRSFRSSI_X
+    // @DisplayName: CRSFRSSI_X
+    // @Description: Horizontal position on screen
+    // @Range: 0 59
+
+    // @Param: CRSFRSSI_Y
+    // @DisplayName: CRSFRSSI_Y
+    // @Description: Vertical position on screen
+    // @Range: 0 21
+    AP_SUBGROUPINFO(crsf_rssi_dbm, "CRSFRSSI", 54, AP_OSD_Screen, AP_OSD_Setting),
```

Comment:
I tend to think we should not lose the _ here. Either all without or with. I guess you could always use XF intead of CRSF

Reply:
Yes, keeping the _ consistent across all panels is much better. I have changed the prefixes to XF so we are able to keep the underscore on all names. But I have changed on the parameters only, everything else (defines, variable names, etc, remain "crsf").